src/main/java/org/folio/jaxb/JAXBDefaultNamespacePrefixMapper.java

```diff
@@ -0,0 +1,35 @@
+package org.folio.jaxb;
+
+import java.util.Map;
+import com.sun.xml.bind.marshaller.NamespacePrefixMapper;
+
+/**
+ * Implementation of {@link NamespacePrefixMapper} that maps the schema
+ * namespaces more to readable names. Used by the jaxb marshaller. Requires
+ * setting the property "com.sun.xml.bind.namespacePrefixMapper" to an instance
+ * of this class.
+ * <p>
+ * Requires dependency on JAXB implementation jars
+ * </p>
+ */
+@SuppressWarnings("squid:S1191") //The com.sun.xml.bind.marshaller.NamespacePrefixMapper is part of jaxb logic
```

Comment:
See https://issues.apache.org/jira/browse/SIS-126

Reply:
Thanks for pointing on solution. 
NameSpacePrefixMapper was deleted